config/locales/modules/personal-social-and-emotional-development.yml

```diff
@@ -202,19 +199,14 @@ en:
 
           $BRAIN
 
-      3-1-1-3b:
```

Comment:
@JPendriss it is essential that you do not delete these. It is how we ensure data integrity and is the reason specs are failing.

Reply:
cc @helenrappand